packages/docusaurus-plugin-content-blog/src/blogUtils.ts

```diff
@@ -100,6 +103,60 @@ function formatBlogPostDate(locale: string, date: Date): string {
   }
 }
 
+type AuthorMap = Record<string, Author>;
+
+async function readAuthorListFile(
+  filePath: string,
+): Promise<AuthorMap | undefined> {
+  const AuthorListSchema = Joi.object<AuthorMap>().pattern(
+    Joi.string(),
+    Joi.object({
+      name: Joi.string(),
+      url: URISchema,
+      imageURL: URISchema,
+      title: Joi.string(),
+    }).rename('image_url', 'imageURL'),
```

Comment:
nice 👍 
I prefer this solution to the more generic one you proposed for other frontmatters because we don't want to reformat too much the user-provided frontmatter, only handle v1 legacy frontmatter names

Reply:
Actually I didn't say my solution is generic 🤪 Just something like `.rename('image_url', 'imageURL').rename('author_title', 'authorTitle')` so that whenever we are using these fields in code we are always using camel case instead of the snake case form

Reply:
:'( that's a bit annoying, I'd rather have good error messages even if this leads to more complexity in Docusaurus